package.json

```diff
@@ -4,16 +4,19 @@
   "description": "Convert react components to native Web Components that work with CanJS.",
   "main": "react-to-can-webcomponent",
   "dependencies": {
-    "can-observation": "^4.2.0"
+    "can-observation": "^4.2.0",
+    "can-observation-recorder": "^1.3.1",
+    "can-queues": "^1.3.1",
+    "can-reflect": "^1.18.0",
+    "ylem": "git+https://github.com/bitovi/ylem.git#hooks"
```

Comment:
Is this needed?

Reply:
No, that was for initial testing.  I will remove that.